first-step-spring-autowired/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.elearning.portfolio</groupId>
+        <artifactId>first-java-project</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>first-step-spring-autowierd</artifactId>
+    <packaging>jar</packaging>
+    <name>Fierst Java Project</name>
+  <description>My first project in Java</description>
+  <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.2.1</version>
+                <configuration>
+                    <mainClass>org.elearning.portfolio.First</mainClass>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.flywaydb</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <version>4.0.3</version>
+                <configuration>
+                    <url>jdbc:h2:file:./first-step/target/portfolio-db</url>
```

Comment:
Tzn. wywalmy to co jest przed target - komendę mvn flyway:migrate będziemy uruchamiać w danym module.